physionet-django/project/templates/project/project_files.html

```diff
@@ -8,11 +8,14 @@
 
 {% block local_js_top %}
 <script src="{% static 'custom/js/cookie.js' %}"></script>
+<script src="{% static 'dropzone/dropzone.min.js' %}"></script>
```

Comment:
I'm getting 404s in the console because dropzone is missing from the static folder.

```
[21/Jan/2022 12:07:35] "GET /static/dropzone/basic.min.css HTTP/1.1" 404 1690
[21/Jan/2022 12:07:35] "GET /static/dropzone/dropzone.min.js HTTP/1.1" 404 1696
[21/Jan/2022 12:07:35] "GET /static/dropzone/dropzone.min.css HTTP/1.1" 404 1699
[21/Jan/2022 12:07:35] "GET /static/dropzone/dropzone.min.js HTTP/1.1" 404 1696
[21/Jan/2022 12:07:36] "GET /static/dropzone/dropzone.min.js HTTP/1.1" 404 1696
[21/Jan/2022 12:07:36] "GET /static/dropzone/dropzone.min.css HTTP/1.1" 404 1699
```

If dropzone is needed, it feels like this dependency (including version details) needs to be captured somewhere in the set up. I don't think it is at the moment?